devicon.json

```
@@ -9738,6 +9738,40 @@
         "color": "#2A0634",
         "aliases": []
     },
+    {
+        "name": "prestashop",
+        "altnames": [],
+        "tags": [
+            "ecommerce",
+            "cms",
+            "php"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
```

font can not use the original versions
```suggestion
                "plain",
                "plain-wordmark"
```